HW_1/task3.py

```diff
@@ -0,0 +1,32 @@
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд
+# и получали билет с номером. Счастливым билетом называют такой билет с шестизначным номером,
+# где сумма первых трех цифр равна сумме последних трех.
+# Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6.
+# Вам требуется написать программу, которая проверяет счастливость билета.
+# *Пример:*
+# 385916 -> yes
+# 123456 -> no
+num = input('Введите шестизначный номер билета: ')
+if len(num) != 6:
+    print(f'Число {num} не корректное для проверки ')
+else:
+    res1 = 0
+    res2 = 0
+    for i in range(len(num)//2):
+        res1 += int(num[i])
+        res2 += int(num[len(num)//2 + i])
```

Comment:
Чтобы не высчитывать срединный индекс можно сделать так:
```suggestion
        res2 += int(num[-1 - i])
```